backend/models/taskServices.js

```diff
@@ -19,11 +21,17 @@ mongoose
   )
   .catch((error) => console.log(error));
 
+let conn;
+
+function connect(newc) {
+    return (conn = newc);
+  }
+
 async function getTasks(task_name, list_name) {
   /* tasks must always have BOTH task AND list name */
   let result;
   if (task_name === undefined && list_name === undefined) {
-    result = await taskModel.find();
```

Comment:
Don't think these changes needed to be made

Reply:
this wasn't a change, just imported the model